test/JDBC/expected/BABEL-5054-vu-verify.out

```diff
@@ -0,0 +1,81 @@
+-- Intermediate JSON result exceeds 4000 characters in length
```

Comment:
1/ can you add test cases which involves declared variables? `declare @obj json ...` 
2/ create a view which is dependent on this and show that OPENJSON is returning correct result